docs/configuration.md

```diff
@@ -743,6 +750,92 @@ Microsoft Teams notifications are sent via the [Incoming Webhooks](https://learn
 [ http_config: <http_config> | default = global.http_config ]
 ```
 
+### `<jira_config>`
+
+JIRA notification are sent via [JIRA Rest API v2](https://developer.atlassian.com/cloud/jira/platform/rest/v2/intro/)
+or [JIRA REST API v3](https://developer.atlassian.com/cloud/jira/platform/rest/v3/intro/#version).
+
+Both APIs have the same feature set. The difference is that V2 uses [Wiki Markup](https://jira.atlassian.com/secure/WikiRendererHelpAction.jspa?section=all)
+for format the issue description and V3 uses [Atlassian Document Format (ADF)](https://developer.atlassian.com/cloud/jira/platform/apis/document/structure/).
```

Comment:
I have no idea. v3 is beta for years.

But I would keep v2 until it gets deprecated/removed:

* Jira On Premise support only v2, because v3 is cloud exclusive
* The body of jira issue in Rest V3 must be in ADF format which no one understand (each paragraph orlink is a nested json property). In v2, Jira Markup is supported.